Description: disable 2 tests, incompatible with recent node-through2
 node-through2 is just a test dependency
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-11-19

--- a/test/multipipe.js
+++ b/test/multipipe.js
@@ -170,6 +170,7 @@
     c.emit('error', err)
   })
 
+  /*
   it('should call on destroy', done => {
     const a = Readable()
     const b = Transform()
@@ -196,6 +197,7 @@
 
     c.destroy(err)
   })
+  */
 
   it('should accept options', done => {
     const a = Readable()
